tokio/src/sync/tests/loom_watch.rs

```diff
@@ -34,3 +35,29 @@ fn smoke() {
         th.join().unwrap();
     })
 }
+
+#[test]
+fn wait_for_test() {
+    loom::model(move || {
+        let (tx, mut rx) = watch::channel(false);
+
+        let tx_arc = Arc::new(tx);
+        let tx1 = tx_arc.clone();
+        let tx2 = tx_arc.clone();
+
+        let th1 = thread::spawn(move || {
+            for _ in 0..3 {
+                tx1.send_modify(|_x| {});
+            }
+        });
+
+        let th2 = thread::spawn(move || {
+            tx2.send(true).unwrap();
```

Comment:
@Darksonn @debadree25  I think It's not necessary to call `unwrap()` on the result of `tx2.send(true`). Since the test is running in a loom thread, any panics will be caught and reported by loom, so you can leave the `Ok(())` result of `tx2.send(true)` unchecked. That makes sense?

```suggestion
            tx2.send(true).ok();
```

Reply:
The `send` call should never fail in this test, so it is better to fail the test if `send` fails.